src/components/IDO/IDO.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Grid, Typography, Box, CardMedia } from '@material-ui/core'
+
+import useStyle from './style'
+import AmountInput from '@components/Inputs/ExchangeAmountInput/ExchangeAmountInput'
+import AnimatedButton from '@components/AnimatedButton/AnimatedButton'
+import icons from '@static/icons'
+
+interface IDOinterface {
+  xBTC: string
+  xETH: string
+  SOL: string
+  USD: string
+  xUSD: string
+  HEADER: string
```

Comment:
props' names should be written in camelCase

Reply:
comment about whole PR

Reply:
done